static/src/javascripts/bootstraps/football.js

```diff
@@ -264,7 +264,7 @@ define([
         });
 
         // Binding
-        bean.on(document.body, 'click', '.js-show-more', function (e) {
+        bean.on(document.body, 'click', '.matches-nav .js-show-more', function (e) {
```

Comment:
So is the issue here that we have more listeners for '.js-show-more' somewhere else and that's what is causing buggy behaviour?

If that's the case then I would suggest to change the name of js hook for this feature instead of increasing specificity this way as there is no indication for `.matches-nav` class that it has other function than just styling thus removing/changing it will return this bug or make show more functionality stop working.